### ISSUE TITLE
Force upload or download changes for complex routes

### ISSUE THREAD

**Issue by swaree** [Zaczero/osm-relatify#73]
Hi, for the last couple of weeks I've been adding some bus routes in my area. This has included some rather complex ones (using the same roundabout multiple times, dead-ends, etc) especially in small towns. For these cases, Relatify ---understandably--- failed to calculate the route and I had to resort to add them by hand.

Rather than suggesting to improve the routing algorithm, I suggest some "Force upload" button or "Download .osm despite errors" to bypass Relatify's proofreading.

**Comment by Zaczero** [comment]
I agree it would be nice to optionally ignore the Relatify checks :+1:
